style.css

```diff
@@ -0,0 +1,214 @@
+@import url('https://fonts.googleapis.com/css2?family=Nabla&family=Poppins&display=swap');
+
+/* minimum mobile size 375px */
+
+@media screen and (min-width: 375px) {
+  body {
+    margin: 0;
+    padding: 0;
+  }
+
+  #header {
+    width: 100%;
+    height: 70vh;
+  }
+
+  .head {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    align-items: center;
+    position: relative;
+  }
+
+  #logo {
+    height: 20px;
+    font-family: 'nabla', cursive;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 200%;
+    line-height: 20px;
+    padding: 0% 5%;
+  }
+
+  .menu-icon {
+    width: 20%;
+  }
+
+  /* section 2 */
+
+  .presentation {
+    background-image: url("headerBackground.svg");
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 85vh;
+    display: flex;
+    width: 100%;
+    flex-direction: column;
+  }
```

Comment:
- [ ] Kindly align your `background-image` to be identical to that of the Figma template that you chose. 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/219465722-7735e867-f7b3-45d4-8e23-4548cc676613.png)

### Figma Design

![Screenshot 2023-02-16 202029](https://user-images.githubusercontent.com/77106962/219465946-1c88bbb2-5321-42de-b147-a660a23d14d8.png)

#### Hint: Pay attention to the background marked orange. Those are the ones that need alignment. You can explore the  `x% and y%` `background-position` values of CSS. https://www.w3schools.com/cssref/pr_background-position.php